[2024-10-08T14:45:16.105+0900] {processor.py:186} INFO - Started process (PID=22279) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:45:16.106+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T14:45:16.107+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:45:16.107+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:45:16.146+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:45:16.146+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T14:45:16.146+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:45:16.146+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T14:45:16.146+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:45:16.266+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.167 seconds
[2024-10-08T14:48:58.487+0900] {processor.py:186} INFO - Started process (PID=22373) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:48:58.488+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T14:48:58.489+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:48:58.489+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:48:58.528+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:48:58.527+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T14:48:58.528+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:48:58.528+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T14:48:58.528+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:48:58.647+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.164 seconds
[2024-10-08T14:52:07.244+0900] {processor.py:186} INFO - Started process (PID=22467) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:52:07.244+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T14:52:07.246+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:52:07.245+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:52:07.286+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:52:07.285+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T14:52:07.286+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:52:07.286+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T14:52:07.286+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:52:07.413+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.175 seconds
[2024-10-08T14:55:12.960+0900] {processor.py:186} INFO - Started process (PID=22591) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:55:12.961+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T14:55:12.963+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:55:12.963+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:55:13.003+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:55:13.002+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T14:55:13.003+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:55:13.003+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T14:55:13.003+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:55:13.129+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.175 seconds
[2024-10-08T14:58:19.675+0900] {processor.py:186} INFO - Started process (PID=22691) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:58:19.676+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T14:58:19.677+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:58:19.677+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:58:19.715+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:58:19.715+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T14:58:19.716+0900] {logging_mixin.py:190} INFO - [2024-10-08T14:58:19.715+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T14:58:19.716+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T14:58:19.873+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.203 seconds
[2024-10-08T15:01:32.419+0900] {processor.py:186} INFO - Started process (PID=22796) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:01:32.420+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:01:32.421+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:01:32.421+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:01:32.461+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:01:32.460+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:01:32.461+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:01:32.461+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:01:32.461+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:01:32.582+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.168 seconds
[2024-10-08T15:04:38.061+0900] {processor.py:186} INFO - Started process (PID=22876) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:04:38.061+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:04:38.062+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:04:38.062+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:04:38.102+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:04:38.102+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:04:38.103+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:04:38.103+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:04:38.103+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:04:38.230+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.174 seconds
[2024-10-08T15:07:43.222+0900] {processor.py:186} INFO - Started process (PID=22975) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:07:43.223+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:07:43.225+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:07:43.225+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:07:43.287+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:07:43.287+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:07:43.288+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:07:43.288+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:07:43.288+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:07:43.425+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.210 seconds
[2024-10-08T15:10:47.389+0900] {processor.py:186} INFO - Started process (PID=23101) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:10:47.389+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:10:47.390+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:10:47.390+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:10:47.427+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:10:47.427+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:10:47.428+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:10:47.427+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:10:47.428+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:10:47.545+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.160 seconds
[2024-10-08T15:13:47.369+0900] {processor.py:186} INFO - Started process (PID=23177) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:13:47.370+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:13:47.371+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:13:47.371+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:13:47.414+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:13:47.413+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:13:47.414+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:13:47.414+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:13:47.414+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:13:47.538+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.175 seconds
[2024-10-08T15:16:47.410+0900] {processor.py:186} INFO - Started process (PID=23268) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:16:47.411+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:16:47.412+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:16:47.411+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:16:47.451+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:16:47.450+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:16:47.451+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:16:47.451+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:16:47.451+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:16:47.575+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.170 seconds
[2024-10-08T15:19:56.706+0900] {processor.py:186} INFO - Started process (PID=23368) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:19:56.707+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:19:56.708+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:19:56.708+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:19:56.751+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:19:56.750+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:19:56.751+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:19:56.751+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:19:56.751+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:19:56.877+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.177 seconds
[2024-10-08T15:23:03.634+0900] {processor.py:186} INFO - Started process (PID=23457) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:23:03.635+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:23:03.636+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:23:03.636+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:23:03.685+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:23:03.684+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:23:03.685+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:23:03.685+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:23:03.685+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:23:03.813+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.185 seconds
[2024-10-08T15:26:09.914+0900] {processor.py:186} INFO - Started process (PID=23539) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:26:09.914+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:26:09.915+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:26:09.915+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:26:09.955+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:26:09.955+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:26:09.955+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:26:09.955+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:26:09.956+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:26:10.080+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.171 seconds
[2024-10-08T15:29:11.139+0900] {processor.py:186} INFO - Started process (PID=23656) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:29:11.139+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:29:11.140+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:29:11.140+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:29:11.178+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:29:11.178+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:29:11.178+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:29:11.178+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:29:11.179+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:29:11.299+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.164 seconds
[2024-10-08T15:32:12.549+0900] {processor.py:186} INFO - Started process (PID=23767) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:32:12.550+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T15:32:12.551+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:32:12.551+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:32:12.590+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:32:12.590+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T15:32:12.591+0900] {logging_mixin.py:190} INFO - [2024-10-08T15:32:12.591+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T15:32:12.591+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T15:32:12.713+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.169 seconds
[2024-10-08T16:37:35.563+0900] {processor.py:186} INFO - Started process (PID=25053) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T16:37:35.564+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T16:37:35.565+0900] {logging_mixin.py:190} INFO - [2024-10-08T16:37:35.565+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T16:37:35.603+0900] {logging_mixin.py:190} INFO - [2024-10-08T16:37:35.603+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T16:37:35.604+0900] {logging_mixin.py:190} INFO - [2024-10-08T16:37:35.604+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T16:37:35.604+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T16:37:35.724+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.165 seconds
[2024-10-08T17:29:08.373+0900] {processor.py:186} INFO - Started process (PID=26421) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:29:08.373+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T17:29:08.374+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:29:08.374+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:29:08.415+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:29:08.414+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T17:29:08.415+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:29:08.415+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T17:29:08.415+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:29:08.539+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.171 seconds
[2024-10-08T17:43:55.764+0900] {processor.py:186} INFO - Started process (PID=26915) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:43:55.765+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T17:43:55.766+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:43:55.766+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:43:55.806+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:43:55.805+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T17:43:55.806+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:43:55.806+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T17:43:55.806+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:43:55.930+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.170 seconds
[2024-10-08T17:56:18.636+0900] {processor.py:186} INFO - Started process (PID=27343) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:56:18.636+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T17:56:18.637+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:56:18.637+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:56:18.675+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:56:18.674+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T17:56:18.675+0900] {logging_mixin.py:190} INFO - [2024-10-08T17:56:18.675+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T17:56:18.675+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T17:56:18.793+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.161 seconds
[2024-10-08T18:01:08.337+0900] {processor.py:186} INFO - Started process (PID=27551) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:01:08.338+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T18:01:08.339+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:01:08.339+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:01:08.378+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:01:08.378+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T18:01:08.379+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:01:08.378+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T18:01:08.379+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:01:08.499+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.167 seconds
[2024-10-08T18:07:16.294+0900] {processor.py:186} INFO - Started process (PID=27781) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:07:16.294+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T18:07:16.295+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:07:16.295+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:07:16.333+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:07:16.332+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T18:07:16.333+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:07:16.333+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T18:07:16.333+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:07:16.453+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.164 seconds
[2024-10-08T18:10:55.432+0900] {processor.py:186} INFO - Started process (PID=27894) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:10:55.433+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T18:10:55.435+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:10:55.434+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:10:55.482+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:10:55.482+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T18:10:55.483+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:10:55.483+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T18:10:55.483+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:10:55.613+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.190 seconds
[2024-10-08T18:14:07.119+0900] {processor.py:186} INFO - Started process (PID=27992) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:14:07.120+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T18:14:07.121+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:14:07.121+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:14:07.159+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:14:07.159+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T18:14:07.159+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:14:07.159+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T18:14:07.159+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:14:07.277+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.163 seconds
[2024-10-08T18:17:16.102+0900] {processor.py:186} INFO - Started process (PID=28085) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:17:16.103+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T18:17:16.104+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:17:16.104+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:17:16.146+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:17:16.146+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T18:17:16.146+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:17:16.146+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T18:17:16.147+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:17:16.269+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.173 seconds
[2024-10-08T18:20:27.724+0900] {processor.py:186} INFO - Started process (PID=28185) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:20:27.725+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T18:20:27.726+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:20:27.726+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:20:27.764+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:20:27.764+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T18:20:27.764+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:20:27.764+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T18:20:27.765+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:20:27.885+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.166 seconds
[2024-10-08T18:47:44.824+0900] {processor.py:186} INFO - Started process (PID=29285) to work on /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:47:44.825+0900] {processor.py:914} INFO - Processing file /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-08T18:47:44.826+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:47:44.826+0900] {dagbag.py:588} INFO - Filling up the DagBag from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:47:44.864+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:47:44.863+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-08T18:47:44.864+0900] {logging_mixin.py:190} INFO - [2024-10-08T18:47:44.864+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-08T18:47:44.864+0900] {processor.py:927} WARNING - No viable dags retrieved from /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-08T18:47:44.981+0900] {processor.py:208} INFO - Processing /Users/alookso/1007MLpj/myenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.161 seconds
